sql/updates/world/3.3.5/2016_04_18_09_world_335.sql

```diff
@@ -0,0 +1,34 @@
+-- Correct gossip_menu_options for NPC 3836 Mountaineer Pebblebitty / Quest 3181
+DELETE FROM `gossip_menu_option` WHERE `menu_id` IN (1201,1202,1203,1204,1205,1206) AND `id` = 0;
+INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`) VALUES
+(1201, 0,0, "What's the worst that could happen?", 4266, 1,1, 1202, 0,0,0,'',0),
+(1202, 0,0, "Another way? Do tell...",             4268, 1,1, 1203, 0,0,0,'',0),
+(1203, 0,0, "Orcs? Badlands? I'm invulnerable!",   4270, 1,1, 1204, 0,0,0,'',0),
+(1204, 0,0, "Absolutely!",                         4272, 1,1, 1205, 0,0,0,'',0),
+(1205, 0,0, "My apologies, Pebblebitty.",          4274, 1,1, 1206, 0,0,0,'',0),
+(1206, 0,0, "Done, done, and done.",               4276, 1,1,    0, 0,0,0,'',0);
```

Comment:
I found them in `broadcast_text` when I started looking for how to add `OptionBroadcastTextID` to the `gossip_menu_option` lines, because I wanted to document it properly and fill in as many available values as possible. I think that might have been the moment when I found out how wrong the original core script text is. I guess I searched for "Pebblebitty" and found out that the nearby lines matched the questions. First, a simple search for "Pebblebitty" in `broadcast_text` :
```sql
SELECT * FROM `broadcast_text`
 WHERE `MaleText` LIKE '%Pebblebitty%' OR `FemaleText` LIKE '%Pebblebitty%';
```
```
    ID  Language  MaleText                                 FemaleText                               EmoteID0  EmoteID1  EmoteID2  EmoteDelay0  EmoteDelay1  EmoteDelay2  SoundId    Unk1    Unk2  VerifiedBuild  
------  --------  ---------------------------------------  ---------------------------------------  --------  --------  --------  -----------  -----------  -----------  -------  ------  ------  ---------------
  4274         0  My apologies, Pebblebitty.               My apologies, Pebblebitty.                      0         0         0            0            0            0        0       0       1          18019
 50742         0  Umm... Pebblebitty... the gate is open.  Umm... Pebblebitty... the gate is open.         0         0         0            0            0            0        0       0       1          18019
```
Then, based on the assumption that the highest `ID` value would be from later years than the lowest value, I started browsing the nearby broadcast_text IDs to see if they contained the rest of the dialogue. Sure enough, I found this :
```sql
SELECT * FROM `broadcast_text` WHERE `ID` BETWEEN 4264 AND 4277;
```
```
    ID  Language  MaleText                             FemaleText                                                                                                                                                                                                                                                                                                                                                                                        EmoteID0  EmoteID1  EmoteID2  EmoteDelay0  EmoteDelay1  EmoteDelay2  SoundId    Unk1    Unk2  VerifiedBuild
------  --------  -----------------------------------  ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------  --------  --------  --------  -----------  -----------  -----------  -------  ------  ------  ---------------
  4265         0                                       I don't care to hear it, $n. Nope, no way are you going to be let into the Searing Gorge. Impossible. Not going to happen. How many times do I have to tell you - I can't let you in! Do you not speak common? Are you an orc in disguise? N-O. NO!$B$BIf I let you in, then I have to let everybody in. Pretty soon your close friends will want in, then their distant relatives, then their fri       0         0         0            0            0            0        0       0       1          18019
  4266         0  What's the worst that could happen?  What's the worst that could happen?                                                                                                                                                                                                                                                                                                                                                                      0         0         0            0            0            0        0       0       1          18019
  4267         0                                       Are you mad? Do you know what manner of beast inhabit that place? The worst that can happen is that we'll have to scoop up your remains with a shovel. We've lost some good dwarves to those awful monsters.$B$BSo the answer is no. If you're bent on getting to the Searing Gorge, you'll have to find another way.                                                                                    0         0         0            0            0            0        0       0       1          18019
  4268         0  Another way? Do tell...              Another way? Do tell...                                                                                                                                                                                                                                                                                                                                                                                  0         0         0            0            0            0        0       0       1          18019
  4269         0                                       I shouldn't even be telling you this but if you can make it past all the critters and beasties in Badlands, you might be able to make it to Searing Gorge.$B$BYes, the Badlands. There's some type of back road the orcs paved near Kargath. Just goes to show you what kind of fools we're dealing with here! A backdoor to hell? What were they thinking?                                              0         0         0            0            0            0        0       0       1          18019
  4270         0  Orcs? Badlands? I'm invulnerable!    Orcs? Badlands? I'm invulnerable!                                                                                                                                                                                                                                                                                                                                                                        0         0         0            0            0            0        0       0       1          18019
  4271         0                                       Invulnerable?                                                                                                                                                                                                                                                                                                                                                                                            0         0         0            0            0            0        0       0       1          18019
  4272         0  Absolutely!                          Absolutely!                                                                                                                                                                                                                                                                                                                                                                                              0         0         0            0            0            0        0       0       1          18019
  4273         0                                       You're a fool that should heed my warnings. No one is invulnerable. You would be wise to remember that bit of advice.                                                                                                                                                                                                                                                                                    0         0         0            0            0            0        0       0       1          18019
  4274         0  My apologies, Pebblebitty.           My apologies, Pebblebitty.                                                                                                                                                                                                                                                                                                                                                                               0         0         0            0            0            0        0       0       1          18019
  4275         0                                       I'll tell you what. Since you fancy yourself the hero type, if you can make your way through the Badlands to the Searing Gorge and bring me back something to prove your heroics, I'll see what I can do about getting you a key.$B$BIn the southeasternmost corner of Searing Gorge is a lady almost as tough as me, Margol the Rager. Maybe you should investigate.                                    0         0         0            0            0            0        0       0       1          18019
  4276         0  Done, done, and done.                Done, done, and done.                                                                                                                                                                                                                                                                                                                                                                                    0         0         0            0            0            0        0       0       1          18019
```
Originally I thought to myself "How can these texts have been sniffed in build 18019 (patch 5.4.7) when the quest was removed in patch 4.0.3a ?", but I settled for the possible explanation that these lines somehow could have been mislabeled as build 18019 by the mass dump to `broadcast_text` and that those lines may have had a different build number before. Maybe some form of mass overwrite happened to the `VerifiedBuild` column at the same time, I don't know.
I'm just guessing. It doesn't make much difference or help us now, though.

So now I have searced Google for clues, I find that the conversation is sort of documented in http://www.wowhead.com/npc=3836/mountaineer-pebblebitty#comments:id=332234 and in http://wow.gamepedia.com/Key_to_Searing_Gorge_quest_chain . I hope those are sufficient.
I used the search term "My apologies, Pebblebitty." to find those links and a couple more.